source/css/_common/components/highlight/highlight.styl

```diff
@@ -64,6 +66,10 @@ pre {
     color: $highlight-foreground;
     line-height: 1em;
     margin-bottom: 1em;
+	margin: 0em;
+	padding: 0.5em;
+	background: #eee;
```

Comment:
https://github.com/theme-next/hexo-theme-next/blob/52d3c024e2e8cb3f1a538ed780187ec8d944f2ed/source/css/_variables/base.styl#L12

Reply:
anything else?

Reply:
to set `border-radius: ?+px`? i don't know the css grammar. 😂

Reply:
Yep. By this variable we can set custom border radius too. See `sidebar.offset` in 3rd example.

> i don't know the css grammar. 

This actually like JS grammar. This is swig. Try it as in examples, it's not hard.

Reply:
yeah, I saw it..